backend/wordsList/letterValues.xml

```diff
@@ -1,575 +1,207 @@
-    <Cell ss:StyleID="S37"><Data ss:Type="Number">2.85</Data>
-    </Cell>
-    <Cell ss:StyleID="S37"><Data ss:Type="Number">1.14</Data>
-    </Cell>
-    <Cell ss:Formula="=20-RC[-2]"><Data ss:Type="Number">17.15</Data>
-    </Cell>
-    <Cell ss:Formula="=20-RC[-2]"><Data ss:Type="Number">18.86</Data>
-    </Cell>
-   </Row>
-   <Row ss:Index="24" ss:Height="12.8">
-    <Cell><Data ss:Type="String">w</Data>
-    </Cell>
-    <Cell ss:StyleID="S37"><Data ss:Type="Number">1.52</Data>
-    </Cell>
-    <Cell ss:StyleID="S37"><Data ss:Type="Number">0.02</Data>
-    </Cell>
-    <Cell ss:Formula="=20-RC[-2]"><Data ss:Type="Number">18.48</Data>
-    </Cell>
-    <Cell ss:Formula="=20-RC[-2]"><Data ss:Type="Number">19.98</Data>
-    </Cell>
-   </Row>
-   <Row ss:Index="25" ss:Height="12.8">
-    <Cell><Data ss:Type="String">x</Data>
-    </Cell>
-    <Cell ss:StyleID="S37"><Data ss:Type="Number">0.04</Data>
-    </Cell>
-    <Cell ss:StyleID="S37"><Data ss:Type="Number">0.22</Data>
-    </Cell>
-    <Cell ss:Formula="=20-RC[-2]"><Data ss:Type="Number">19.96</Data>
-    </Cell>
-    <Cell ss:Formula="=20-RC[-2]"><Data ss:Type="Number">19.78</Data>
-    </Cell>
-   </Row>
-   <Row ss:Index="26" ss:Height="12.8">
-    <Cell><Data ss:Type="String">y</Data>
-    </Cell>
-    <Cell ss:StyleID="S37"><Data ss:Type="Number">0.04</Data>
-    </Cell>
-    <Cell ss:StyleID="S37"><Data ss:Type="Number">1.01</Data>
-    </Cell>
-    <Cell ss:Formula="=20-RC[-2]"><Data ss:Type="Number">19.96</Data>
-    </Cell>
-    <Cell ss:Formula="=20-RC[-2]"><Data ss:Type="Number">18.99</Data>
-    </Cell>
-   </Row>
-   <Row ss:Index="27" ss:Height="12.8">
-    <Cell><Data ss:Type="String">z</Data>
-    </Cell>
-    <Cell ss:StyleID="S37"><Data ss:Type="Number">1.39</Data>
-    </Cell>
-    <Cell ss:StyleID="S37"><Data ss:Type="Number">0.47</Data>
-    </Cell>
-    <Cell ss:Formula="=20-RC[-2]"><Data ss:Type="Number">18.61</Data>
-    </Cell>
-    <Cell ss:Formula="=20-RC[-2]"><Data ss:Type="Number">19.53</Data>
-    </Cell>
-   </Row>
-  </ss:Table>
-  <WorksheetOptions xmlns="urn:schemas-microsoft-com:office:excel">
-   <PageSetup>
-    <Layout x:StartPageNumber="1" />
-    <Header x:Margin="0.7875" x:Data="&amp;C&amp;&quot;Times New Roman,Regular&quot;&amp;12&amp;A" />
-    <Footer x:Margin="0.7875" x:Data="&amp;C&amp;&quot;Times New Roman,Regular&quot;&amp;12Page &amp;P" />
-    <PageMargins x:Top="1.05277777777778" x:Bottom="1.05277777777778" x:Left="0.7875" x:Right="0.7875" />
-   </PageSetup>
-   <FitToPage />
-   <Print>
-    <ValidPrinterInfo />
-    <PaperSizeIndex>9</PaperSizeIndex>
-   </Print>
-   <Panes>
-    <Pane>
-     <Number>3</Number>
-     <ActiveRow>22</ActiveRow>
-     <ActiveCol>11</ActiveCol>
-     <RangeSelection>R23C12</RangeSelection>
-    </Pane>
-   </Panes>
-  </WorksheetOptions>
- </Worksheet>
-</Workbook>
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<LetterValues xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">

```

Comment:
I'm sorry but these are not the correct values! the correct ones are in @Raque10bar backend (as explained). These contain more letters for all languages